lang/po/new/sv.po

```diff
@@ -1,1772 +1,1758 @@
+
+# MSG_EJECT_REMOVE c=20 r=4
+#: mmu.cpp:1421
+msgid "Please remove filament and then press the knob."
+msgstr "Ta bort filamentet och tryck sedan pa knappen."
+
+# MSG_REMOVE_STEEL_SHEET c=20 r=4
+#: messages.c:84
+msgid "Please remove steel sheet from heatbed."
+msgstr "Ta bort metallskivan fran v\xe1rmeb\xe1dden."
+
+# MSG_RUN_XYZ c=20 r=4
+#: Marlin_main.cpp:5338
+msgid "Please run XYZ calibration first."
+msgstr "Utf\xefr XYZ-kalibrering f\xefrst."
+
+# MSG_UPDATE_MMU2_FW c=20 r=4
+#: mmu.cpp:1341
+msgid "Please update firmware in your MMU2. Waiting for reset."
+msgstr "Uppdatera firmware i din MMU2. V\xe1ntar pa aterst\xe1llning."
+
+# MSG_PLEASE_WAIT c=20
+#: messages.c:76
+msgid "Please wait"
+msgstr "V\xe1nligen v\xe1nta"
+
+# MSG_REMOVE_SHIPPING_HELPERS c=20 r=3
+#: ultralcd.cpp:4957
+msgid "Please remove shipping helpers first."
+msgstr "V\xe1nligen ta bort fraktinsatserna f\xefrst."
+
+# MSG_PREHEAT_NOZZLE c=20
+#: messages.c:78
+msgid "Preheat the nozzle!"
+msgstr "F\xefrv\xe1rm munstycket!"
+
+# MSG_PREHEAT c=18
+#: ultralcd.cpp:6655
+msgid "Preheat"
+msgstr "F\xefrv\xe1rm"
+
+# MSG_WIZARD_HEATING c=20 r=3
+#: messages.c:119
+msgid "Preheating nozzle. Please ewait."
+msgstr "F\xefrv\xe1rmer munstycke. V\xe1nta."
+
+# MSG_NEW_FIRMWARE_PLEASE_UPGRADE c=20
+#: util.cpp:199
+msgid "Please upgrade."
+msgstr "V\xe1nligen uppgradera."
+
+# MSG_PRESS_TO_PREHEAT c=20 r=4
+#: Marlin_main.cpp:12049
+msgid "Press the knob to preheat nozzle and continue."
+msgstr "Tryck pa knappen f\xefr att f\xefrv\xe1rma munstycket och forts\xe1tta."
+
+# MSG_FS_PAUSE c=5
+#: fsensor.cpp:730
+msgid "Pause"
+msgstr "Paus"
+
+# MSG_POWER_FAILURES c=15\xe1\xe1zzz
+#: messages.c:77
+msgid "Power failures"
+msgstr "Str\xefmavbrott"
+
+# MSG_PRINT_ABORTED c=20
+#: messages.c:80
+msgid "Print aborted"
+msgstr "Utskriften avbr\xefts"
+
+# MSG_PREHEATING_TO_LOAD c=20
+#: ultralcd.cpp:2297
+msgid "Preheating to load"
+msgstr "F\xefrv\xe1rmer f\xefr laddn"
+
+#MSG_PREHEATING_TO_UNLOAD c=20
+#: ultralcd.cpp:2302
+msgid "Preheating to unload"
+msgstr "F\xefrv\xe1rmer for utmatn"
+
+# MSG_SELFTEST_PRINT_FAN_SPEED c=18
+#: ultralcd.cpp:8501
+msgid "Print fan:"
+msgstr "Utskriftsfl\xe1kt:"
+
+# MSG_CARD_MENU c=18
+#: messages.c:22
+msgid "Print from SD"
+msgstr "Skriv ut fran SD"
+
+# MSG_PRESS_KNOB c=20
+#: ultralcd.cpp:2130
+msgid "Press the knob"
+msgstr "Tryck pa knappen"
+
+# MSG_PRINT_PAUSED c=20
+#: ultralcd.cpp:907
+msgid "Print paused"
+msgstr "Utskriften pausad"
+
+# MSG_RESUME_NOZZLE_TEMP c=20 r=4
+#: mmu.cpp:726
+msgid "Press the knob to resume nozzle temperature."
+msgstr "Tryck pa knappen f\xefr att aterst\xe1lla munstyckstemperaturen."
+
+# MSG_FOLLOW_CALIBRATION_FLOW c=20 r=8
+#: messages.c:46
+msgid "Printer has not been calibrated yet. Please follow the manual, chapter First steps, section Calibration flow."
+msgstr "Skrivaren har inte kalibrerats \xe1nnu. V\xe1nligen f\xeflj manualen, kapitel F\xefrsta stegen, avsnitt Kalibreringsfl\xefde."
```

Comment:
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.